### DISCUSSION TITLE
Does the automatic installer work with 3.2? Next gen is crashing my computer.

### DISCUSSION THREAD

**Discussion by ZukoIsBack** [Aelto/W3_RandomEncounters_Tweaks#191] [Q&A] [answered]
I don't want to screw it up, so I thought I'd ask beforehand.

**Comment by Aelto** [comment, marked as answer]
Which installer? And what do you mean by 3.2? If you wish to get the mod in Old Gen (1.32) then you'd have to use the old-gen installer or do it manually. All documentation for old gen was removed so i'd recommend using the install script.

You can get the old gen install script [here](https://github.com/Aelto/tw3-random-encounters-reworked/blob/master/scripts/install-mod.oldgen.ps1), click the download icon then use it like the regular install script.

_download icon_:
![image](https://user-images.githubusercontent.com/13685633/226187709-69b96530-aff4-46b9-9aaa-9f1a0a267755.png)

**Reply by ZukoIsBack** [reply]
Thank you! Yes, I meant Old Gen 1.32. I thought it was 3.2 for some reason. Thank you so much!
